pkg/cloud/openstack/clients/networkservice.go

```diff
@@ -320,3 +382,15 @@ func (s *NetworkService) getNetworkByName(networkName string) (networks.Network,
 	}
 	return networks.Network{}, errors.New("too many resources")
 }
+
+func (s *NetworkService) CreateFloatingIP(floatingNetworkID string) (string, error) {
+	opts := floatingips.CreateOpts{
+		FloatingNetworkID: floatingNetworkID,
```

Comment:
As mentioned. Set a Description here with which you can find the floating ip in case of a restart of the controller. Maybe `machine.Meta.Uid` does work!? Otherwise `namespace/cluster.Name/machine.Name` would be fine.